src/test/kotlin/apply/ApplicationFormFixtures.kt

```diff
@@ -1,54 +1,71 @@
 package apply
 
+import apply.application.AnswerRequest
 import apply.domain.applicationform.ApplicationForm
-import apply.domain.recruitmentitem.Answer
-import apply.domain.recruitmentitem.Answers
+import apply.domain.applicationform.ApplicationFormAnswer
+import apply.domain.applicationform.ApplicationFormAnswers
+
+private const val EXCESS_LENGTH = 2000
 
 fun createApplicationForm(
     applicantId: Long = 1L,
     recruitmentId: Long = 1L,
     referenceUrl: String = "http://example.com",
-    answers: Answers = createAnswers()
+    applicationFormAnswers: ApplicationFormAnswers = createApplicationFormAnswers()
 ): ApplicationForm {
-    return ApplicationForm(applicantId, recruitmentId, referenceUrl, answers)
+    return ApplicationForm(applicantId, recruitmentId, referenceUrl, applicationFormAnswers)
 }
 
-fun createAnswers(
-    answers: MutableList<Answer> = mutableListOf(
-        createAnswer(),
-        createAnswer("책임감", 2L)
+fun createApplicationFormAnswers(
+    applicationFormAnswers: MutableList<ApplicationFormAnswer> = mutableListOf(
+        createApplicationFormAnswer(),
+        createApplicationFormAnswer("책임감", 2L)
     )
-): Answers {
-    return Answers(answers)
+): ApplicationFormAnswers {
+    return ApplicationFormAnswers(applicationFormAnswers)
 }
 
-fun createAnswer(
+fun createApplicationFormAnswer(
     contents: String = "스타트업을 하고 싶습니다.",
     recruitmentItemId: Long = 1L
-): Answer {
-    return Answer(contents, recruitmentItemId)
+): ApplicationFormAnswer {
+    return ApplicationFormAnswer(contents, recruitmentItemId)
+}
+
+fun createAnswerRequest(
+    contents: String = "책임감",
+    recruitmentItemId: Long = 1L
+): AnswerRequest {
+    return AnswerRequest(contents, recruitmentItemId)
+}
+
+fun createExcessOfLengthAnswerRequest(
+    contents: String = "*".repeat(EXCESS_LENGTH),
```

Comment:
```suggestion
    contents: String = "*".repeat(MAXIMUM_LENGTH + 1),
```
RecruitmentFixtures에 있는 변수를 재활용해도 좋을 것 같아요